tests/test_python.py

```diff
@@ -96,18 +97,24 @@ def test_python_webserver_1(container_per_test):
     ).is_listening
 
     # start of the python http server
-    bci_pyt_serv = container_per_test.connection.run_expect(
-        [0], f"timeout 240s python3 -m http.server {port} &"
-    ).stdout
+    container_per_test.connection.run_expect(
+        [0], f"python3 -m http.server {port} &"
```

Comment:
Another option would be to modify the entrypoint of `CONTAINER_IMAGES_T` to `["/usr/bin/python3",  "-m", "http.server"]` and add a `HEALTHCHECK` that will check whether the server is running. Then `pytest_container` will do the waiting for you.